va/va.c

```diff
@@ -1154,6 +1159,7 @@ vaQuerySurfaceAttributes(
 
     VA_TRACE_LOG(va_TraceQuerySurfaceAttributes, dpy, config, attrib_list, num_attribs);
     VA_TRACE_RET(dpy, vaStatus);
+    VA_TRACE_PA(dpy, QUERY_SURFACE_ATTR, TRACE_END, num_attribs, attrib_list);
```

Comment:
P short for pointer. caller can pass variable pointer to add variable to trace data